README.md

```diff
@@ -1392,4 +1392,9 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
 
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
```

Comment:
Please don't make changes in previous code, it'll make conflicts for you and another students.
And also could you check image path? 
<img width="809" alt="Screenshot 2020-10-13 at 15 17 20" src="https://user-images.githubusercontent.com/14895118/95859483-5d1e6e00-0d67-11eb-8af9-5ce53df0dbac.png">